invenio_vocabularies/services/service.py

```diff
@@ -105,7 +107,7 @@ def create_type(self, identity, id, pid_type):
         db.session.commit()
         return type_
 
-    def search(self, identity, params=None, es_preference=None, type=None,
+    def search(self, identity, type, params=None, es_preference=None,
```

Comment:
This will break with this change https://github.com/inveniosoftware/invenio-vocabularies/blob/master/invenio_vocabularies/resources/resource.py#L106

Now the question is why the tests don't break 😄 

To avoid problems before release, please revert this change

Reply:
Tests do not break because the call is made with "kwargs" i.e. type=... not just assuming the "position". But yes, ill revert.